be/src/formats/parquet/file_reader.cpp

```diff
@@ -515,31 +589,32 @@ int32_t FileReader::_get_partition_column_idx(const std::string& col_name) const
     return -1;
 }
 
-Status FileReader::_decode_min_max_column(const ParquetField& field, const std::string& timezone,
-                                          const TypeDescriptor& type, const tparquet::ColumnMetaData& column_meta,
-                                          const tparquet::ColumnOrder* column_order, ColumnPtr* min_column,
-                                          ColumnPtr* max_column) const {
-    DCHECK_EQ(field.physical_type, column_meta.type);
+Status FileReader::_get_min_max_value(const SlotDescriptor* slot, const tparquet::ColumnMetaData* column_meta,
+                                      const ParquetField* field, std::vector<std::string>& min_values,
+                                      std::vector<std::string>& max_values) const {
+    // When statistics is empty, column_meta->__isset.statistics is still true,
+    // but statistics.__isset.xxx may be false, so judgment is required here.
+    bool is_set_min_max = (column_meta->statistics.__isset.max && column_meta->statistics.__isset.min) ||
+                          (column_meta->statistics.__isset.max_value && column_meta->statistics.__isset.min_value);
+    if (!is_set_min_max) {
+        return Status::Aborted("No exist min/max");
+    }
 
-    auto sort_order = sort_order_of_logical_type(type.type);
-    if (!_has_correct_min_max_stats(column_meta, sort_order)) {
+    DCHECK_EQ(field->physical_type, column_meta->type);
+    auto sort_order = sort_order_of_logical_type(slot->type().type);
+
+    if (!_has_correct_min_max_stats(*column_meta, sort_order)) {
         return Status::Aborted("The file has incorrect order");
     }
 
-    std::vector<std::string> min_value;
-    std::vector<std::string> max_value;
-
-    if (column_meta.statistics.__isset.min_value) {
-        min_value.emplace_back(column_meta.statistics.min_value);
-        max_value.emplace_back(column_meta.statistics.max_value);
+    if (column_meta->statistics.__isset.min_value) {
+        min_values.emplace_back(column_meta->statistics.min_value);
```

Comment:
Why use std::vector，Is it possible to have multiple values ​​in the vector?

Reply:
Yes, it's reserved for pageindex, there we have minvalues and maxvalues in vector.